src/llmq/quorums_signing.cpp

```diff
@@ -562,7 +562,7 @@ CQuorumCPtr CSigningManager::SelectQuorumForSigning(Consensus::LLMQType llmqType
     uint256 startBlock;
     {
         LOCK(cs_main);
-        if (signHeight > chainActive.Height()) {
+        if (signHeight - SIGN_HEIGHT_OFFSET > chainActive.Height()) {
```

Comment:
Let's maybe introduce `int startBlockHeight = signHeight - SIGN_HEIGHT_OFFSET;`? (and rename  `startBlock` to `startBlockHash` while at it)